test/worker/test-worker.ts

```diff
@@ -0,0 +1,41 @@
+import test from "ava";
+import * as sinon from "sinon";
+import { WorkerWrapperImpl } from "@worker/worker";
+import Storage from "@worker/storage";
+import { StoredWordList } from "@worker/storage";
+import * as compiler from "@predictive-text-studio/lexical-model-compiler";
+
```

Comment:
I tried to mock storage and test saveFile as well and File doesn't exist on node xD

Reply:
Yeah :/